daemon/volumes.go

```diff
@@ -29,7 +29,7 @@ type mountPoint struct {
 	RW          bool
 	Volume      volume.Volume `json:"-"`
 	Source      string
-	Relabel     string
+	Mode        string
```

Comment:
Why not have both fields and just fallback when `Mode` is empty? This way we can phase out `Relabel`.